extensions/amp-story/0.1/amp-story.css

```diff
@@ -194,8 +194,8 @@ amp-story[standalone]:fullscreen {
   display: none !important;
 }
 
-.audio-playing:not([muted]) .i-amphtml-story-mute-audio-control,
-.audio-playing[muted] .i-amphtml-story-unmute-audio-control {
+.has-audio:not([muted]) .i-amphtml-story-mute-audio-control,
```

Comment:
Just a heads up: this rename is technically a breaking API change, since publishers can write styles depending on the name (since it doesn't begin with `i-amphtml-`)

I think it's okay, but we'll need to keep an eye out in case people were depending on it.

Reply:
Ah, right! I reverted this change. If that works with you, this should be part of a larger refactoring around the sound indicators? From what we discussed, if we want to add an equalizer, we'll need ``has-audio``, ``audio-playing``, that way it'll be a non breaking API change.